modules/combined/test/tests/internal_volume/tests

```diff
@@ -28,13 +33,15 @@
     cli_args = 'Postprocessors/internalVolume/component=1 Outputs/file_base=rz2_sm_out'
     csvdiff = 'rz2_sm_out.csv'
     prereq = 'rz_sm'
+    requirement = 'The system shall compute internal volume given an enclosing sideset for axisymmetric coordinates and with the y coordinate for solid_mechanics'
```

Comment:
Seems like we should say what the y-coordinate is being used for in the test.  Something like "... with the y coordinate used in the integration over the sideset..."?

Reply:
Done.